backends-clickhouse/src/test/scala/io/glutenproject/execution/GlutenClickHouseTPCHParquetSuite.scala

```diff
@@ -660,14 +660,15 @@ class GlutenClickHouseTPCHParquetSuite extends GlutenClickHouseTPCHAbstractSuite
     runQueryAndCompare(
       // l_shipinstruct: DELIVER IN PERSON
       // l_returnflag: N
-      // l_linenumber: 1
       """
         |select position('D', l_shipinstruct, 0), position('', l_shipinstruct, 0),
         |position('I', l_shipinstruct, 5), position('IN', l_shipinstruct),
         |position('', l_shipinstruct), locate(l_returnflag, l_shipinstruct),
         |position(l_returnflag in l_shipinstruct), position('bar', 'foobarbar'),
         |position(l_returnflag, 'TENSTNTEST', 4), position('bar', 'foobarbar', 5),
-        |position(l_returnflag, l_shipinstruct, l_linenumber + 11)
+        |position(l_returnflag, l_shipinstruct, 12),
```

Comment:
'l_linenumber + 11' here is intended to ensure that start_pos is not a constant column

Reply:
Done